test/lib/usd/translators/testUsdExportBindTransform.py

```diff
@@ -0,0 +1,60 @@
+#!/pxrpythonsubst
```

Comment:
Since it’s a new file, please apply change following #1026

Reply:
Resolved in 71d7bb0.